official-antistasi-community/A3-Antistasi
# Phase 1 of mod integration safety work.

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    Re-formats the Template folder to make it more organised and readable.
    Updates file paths for selector.sqf
    Updates file call for the PvP loadout funtion.
    Updates the template guide txt
    Adds full set of example templates for integrator to read.

### Please specify which Issue this PR Resolves.
closes #1266 

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in singleplayer?
2. [x] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
    Load all modsets on 1 of each environment of map, make sure that *all* pvp loadouts load properly and that the correct template is loaded and used.
********************************************************
Notes:
  This was the 2nd easiest part.
